iquhack/templates/iquhack/iquhack.html

```diff
@@ -0,0 +1,168 @@
+{% extends "home/base.html" %}
+{% load static %}
+
+{% block master_title %}
+    iQuHACK
+{% endblock %}
+
+{% block extend_head %}
+<meta property="twitter:title" content="iQuHACK" />
+<meta property="og:title" content="iQuHACK" />
+
+<meta property="twitter:description" content="iQuHACK (interdisciplinary Quantum HACKathon) is MIT's annual quantum hackathon.">
+<meta property="og:description" content="iQuHACK (interdisciplinary Quantum HACKathon) is MIT's annual quantum hackathon." />
+
+<meta property="twitter:image" content="https://www.iquise.mit.edu{% static 'iquhack/images/schrodingersduck.png' %}">
+<meta property="og:image" content="https://www.iquise.mit.edu{% static 'iquhack/images/schrodingersduck.png' %}" />
+
+<link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Teko&display=swap" rel="stylesheet">
+
+<link rel="stylesheet" href="{% static 'iquhack/assets/css/custom.css' %}" type="text/css"></style>
+
+<style type="text/css">
+.sponsor.t-platform img {
+    height: {{ hackathon.logo_max_height }}px;
+    margin: {{ hackathon.logo_max_side_margin }}px {{ hackathon.logo_max_side_margin }}px {{ hackathon.logo_max_bottom_margin }}px;
+}
+{% for tier, height, side, bottom, tier_sponsors in sponsors %}
+.sponsor.t{{ tier.index }} img {
+    height: {{ height }}px;
+    margin: {{ side }}px {{ side }}px {{ bottom }}px;
+}
+{% endfor %}
+</style>
+{% endblock %}
+
+{% block more_js %}
+<script src="{% static 'website/assets/js/jquery.min.js' %}"></script>
+{% endblock %}
+
+{% block banner %}
+<div id="banner-container" class="iquhack">
+    <img class="background" alt="Banner" src="{{ hackathon.back_drop_image.url }}">
+    {% if hackathon.open %}
+    <div id="banner" class="open" onclick="location.href='{{ hackathon.link }}';">
+    {% else %}
+    <div id="banner" class="">
+    {% endif %}
+        <p><strong>MIT QUANTUM HACKATHON</strong></p>
+        <div class="logo">
+            <img alt="iQuHACK" src="{% static 'iquhack/images/logo.svg' %}">
+            <h1>{{ hackathon.start_date| date:'Y' }}</h1></div>
+        <p><strong><i>{{ formatted_event_date }}</i></strong></p>
+        {% if hackathon.early %}
+            <p><span>Registration Opens on </span><span>{{ hackathon.opens | date:'l, F jS (P e)' }}</span></p>
+            {% if hackathon.early_note %}
+            <p>{{ hackathon.early_note }}</p>
+            {% endif %}
+        {% elif hackathon.open %}
+            <p><span><strong>Click to Register</strong> by </span><span>{{ hackathon.deadline | date:'l, F jS (P e)' }}</span></p>
```

Comment:
space after 'by' doesn't render

Reply:
Might need to add css to add 1 char padding-right to all but last span in a p element.